requirements/base.txt

```diff
@@ -21,6 +21,7 @@ newspaper3k==0.2.8
 https://github.com/DemocracyClub/dc_signup_form/archive/refs/tags/2.1.1.tar.gz
 https://github.com/DemocracyClub/design-system/archive/refs/tags/0.2.1.tar.gz
 https://github.com/DemocracyClub/dc_django_utils/archive/refs/tags/0.1.8.tar.gz
+git+https://github.com/DemocracyClub/dc_logging.git@0.0.9
```

Comment:
I think there was a reason we used the `tar.gz` archives above this line...was it to do with when we dump `pipenv` in to a `requirements.txt` file? If this fails, maybe change it to the other format (I'm interested to know!)